src/current/_includes/v24.1/known-limitations/aost-limitations.md

```diff
@@ -0,0 +1 @@
+CockroachDB does not support placeholders in [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}). The time value must be embedded in the SQL string. [#30955](https://github.com/cockroachdb/cockroach/issues/30955)
```

Comment:
> The time value must be embedded in the SQL string.

optional nit: I would call this a "constant", so you could say something like "The time value must be a constant value embedded in the SQL string" or "The time value must be embeded in the SQL string as a constant value".